### PR TITLE
started outlining db model, schema, and endpoints for events

### PR DESCRIPTION
**Pull request by morden35**
Hey James, here is a (very rough) start to the api work for events.

1. I started with an Event() class in db/models/events.py. Should I add more columns to Event()? 

2. I also created api/events.py. This is basically a copy of api/committees.py but I replaced Committee with Event and commented out lines that I don't think are needed anymore.

3. In schemas.py I added an Event() class. Should I also add EventClassification()?

@jamesturk

### CODE REVIEW

File: api/db/models/events.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from .. import Base
+from sqlalchemy import Column, String, ForeignKey, Integer, Boolean, DateTime
+
+
+class Event(Base):
+    __tablename__ = "opsencivicdata_event"
+
+    id = Column(String, primary_key=True, index=True)
+    name = Column(String)
+    jurisdiction = ColumnString(String)
```
Comment (jamesturk) [inline api/db/models/events.py:10]: typo on this one ColumnString

File: api/db/models/events.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from .. import Base
+from sqlalchemy import Column, String, ForeignKey, Integer, Boolean, DateTime
+
+
+class Event(Base):
+    __tablename__ = "opsencivicdata_event"
```
Comment (jamesturk) [inline api/db/models/events.py:6]: typo here

File: api/events.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    include: List[EventInclude] = Query(
+        [], description="Additional includes for the Event response."
+    ),
+    db: SessionLocal = Depends(get_db),
+    auth: str = Depends(apikey_auth),
+):
+    """Get details on a single event by ID."""
+    query = db.query(models.Organization).filter(
+        models.Organization.id == event_id,
+        models.Organization.classification.in_("event"),
```
Comment (jamesturk) [inline api/events.py:106]: can remove this

**Comment by morden35** [comment]
@jamesturk 

1. Should I use from sqlalchemy_utils import URLType for EventLocation.url?

2. What is is MimetypeLinkBase?

**Comment by jamesturk** [comment]
1. for URLs you can use the normal string/char type :)
2. https://github.com/openstates/openstates-core/blob/main/openstates/data/models/base.py#L83 adds these fields
3. I'm not sure actually, it is OK to leave off the back-populates for now, it's just a warning AFAIK & we can sort it out later.

**Comment by jamesturk** [comment]
folded this into #15 WIP

**Pull request closed without merging**